shapes.py

```diff
@@ -0,0 +1,29 @@
+class Shape:
+    pass
```

Comment:
You created the base class but you didn't inherit the base class from the other shapes.